apple/xcode/ios/Outline/Outline-Info.plist

```diff
@@ -57,6 +57,9 @@
 	<key>UISupportedInterfaceOrientations~ipad</key>
 	<array>
 		<string>UIInterfaceOrientationPortrait</string>
+		<string>UIInterfaceOrientationPortraitUpsideDown</string>
```

Comment:
@alalamav is this code auto generated by Cordova?
Maybe https://github.com/Jigsaw-Code/outline-client/pull/595 is enough?